### PR TITLE
Kaduk IESG feedback

### PR DESCRIPTION
**Pull request by martinthomson**
Lots of small changes here, though nothing of any real consequence, I believe.

### CODE REVIEW

File: draft-ietf-httpbis-http2bis.xml [hunk 1]
```diff
@@ -309,9 +309,10 @@
                 document do not hold unless TLS is used; see <xref target="security"/>.
             </t>
             <t>
-                The "h2c" string was previously used as a token for use in the HTTP Upgrade mechanism's
-                Upgrade header field (<xref target="HTTP" section="7.8"/>). This usage was never widely
-                deployed, and is deprecated by this document.
+                The "h2c" string was previously used as a token for use in the HTTP Upgrade
```
Comment (mnot) [inline draft-ietf-httpbis-http2bis.xml:312]:
```suggestion
                The "h2c" string was previously used in the HTTP Upgrade
```

File: draft-ietf-httpbis-http2bis.xml [hunk 2]
```diff
@@ -538,9 +539,10 @@ HTTP Frame {
           processes as header compression and decompression.
         </t>
         <t>
-          Field blocks carry the compressed bytes of a field section, with header sections
-          also carrying control data associated with the message in the form of <xref target="PseudoHeaderFields">pseudo-header fields</xref> that use the same format as a
-          field line.
+          A field section consists of exactly one field block.  Each field block carries the
+          compressed bytes of a single field section.  Header sections also include control data
```
Comment (Lukasa) [inline draft-ietf-httpbis-http2bis.xml:543]: Hmm, this section reads strangely to me. Field sections are made up of field blocks which carry the compressed bytes of field sections? This feels a little recursive.

Do we have a clearer way to express this idea?

Comment (martinthomson) [reply draft-ietf-httpbis-http2bis.xml:543]: I gave it a try.  I had to submit a version, so I went with this:

>          Each field block carries all of the compressed field lines of a single field section.

Comment (Lukasa) [reply draft-ietf-httpbis-http2bis.xml:543]: I think that would work much better.

**Comment by martinthomson** [comment]
Merged manually.

**Pull request closed without merging**